examples/vanilla-ts-esm/public/mux-player-error.html

```diff
@@ -41,7 +41,7 @@ <h1><a href="/">Elements</a></h1>
     <!-- Correct playbackid: ihZa7qP1zY8oyLSQW9TS602VgwQvNdyIvlk9LInEGU2s -->
     <mux-player
       stream-type="on-demand"
-      playback-id="Sc89iWAyNkhJ3P1rQ02nrEdCFTnfT01CZ2KmaEcxXfB00"
+      playback-id="ihZa7qP1zY8oyLSQW9TS602VgwQvNdyIvlk9LInEGU2"
```

Comment:
question(blocking): Should this have been committed? (name is "mux-player-error.html" so just confirming here)

Reply:
yes, this is a playback id with the last character left off so it errors out. I just didn't catch this last time when I did a project wide search/replace of the playback id starting with `Sc89iWAyNkhJ3P1rQ02nrEdCFTnfT01CZ2KmaEcxXfB00`

you can see above the complete playback id was indeed replaced.